linode/manager
# refactor: [M3-8909] - Migrate `EditableText` to `ui` package

## Description 📝
Moves the `EditableText ` component to the `@linode/ui` package and updates existing imports.

## Changes  🔄
- Move `EditableText ` component to `@linode/ui` package
- Update existing imports
- Added remaining react eslint rules and specific rules related to this component/story

## Target release date 🗓️
N/A

## How to test 🧪
- Confirm all checks pass
- Confirm all the necessary imports are updated
- Confirm nothing on CM and Storybook breaks
   - Ensure everything works as expected wherever `EditableText` is used
   - Run `rm -r packages/manager/node_modules/.cache`  + clear your browser cache before testing Storybook

## As an Author, I have considered 🤔

- 👀 Doing a self review
- ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- 🤏 Splitting feature into small PRs
- ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- 🧪 Providing/improving test coverage
- 🔐 Removing all sensitive information from the code and PR description
- 🚩 Using a feature flag to protect the release
- 👣 Providing comprehensive reproduction steps
- 📑 Providing or updating our documentation
- 🕛 Scheduling a pair reviewing session
- 📱 Providing mobile support
- ♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅
